packages/loaders/src/Skeleton.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+import { transparentize } from 'polished';
+import { retrieveTheme } from '@zendeskgarden/react-theming';
+import {
+  zdColorGrey800,
+  zdColorGrey100,
+  zdColorKale800,
+  zdColorWhite,
+  zdSpacingXxs
+} from '@zendeskgarden/css-variables';
+
+const COMPONENT_ID = 'loaders.skeleton';
+
+const fadeInAnimation = props => keyframes`
+  0% {
+    opacity: 0;
+  }
+
+  60% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: ${props.dark ? '0.15' : '0.1'};
+  }
+`;
+
+const skeletonAnimation = keyframes`
+  0% {
+    left: -1200px;
+  }
+
+  100% {
+    left: 100%;
+  }
+`;
+
+const StyledSkeleton = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  display: inline-block;
+  position: relative;
+
+  animation: ${fadeInAnimation} 700ms linear; /* stylelint-disable-line */
+  opacity: ${props => (props.dark ? '0.15' : '0.1')};
+
+  border-radius: ${zdSpacingXxs};
+  background-color: ${props => (props.dark ? zdColorWhite : zdColorGrey800)};
```

Comment:
You could simplify this animation a bit and set an rgba.

```js
import { rgba } from 'polished';

const StyledSkeleton = styled.div`
  background-color: ${props => (props.dark ? rgba(zdColorWhite, 0.1) : rgba(zdColorGrey800, 0.15))};
`;
```

You could then drop setting opacity and just have the animation set what the 100% keyframe is:

```js
const fadeInAnimation = props => keyframes`
 100% {
    opacity: ${props.dark ? '0.15' : '0.1'};
  }
`;